src/Illuminate/Http/Concerns/InteractsWithInput.php

```diff
@@ -283,6 +283,20 @@ public function boolean($key = null, $default = false)
         return filter_var($this->input($key, $default), FILTER_VALIDATE_BOOLEAN);
     }
 
+    /**
+     * Decode input from JSON to array/object. Useful when only parts are json encoded.
+     *
+     * Returns decoded JSON as array or object
+     *
+     * @param string $key
+     * @param bool $assoc
+     * @param int $options
```

Comment:
Please add a return type and use the same DocBlock styling as in the rest of the framework (two spaces before and after the type for param annotations).